queries/authzed/highlights.scm

```diff
@@ -0,0 +1,42 @@
+; highlights.scm
+((identifier) @keyword
+ (#match? @keyword "^(definition|permission|relation)$"))
+
+((permission_literal) @variable.builtin)
+
+(permission (identifier) @type)
+(relation (identifier) @constant)
+(perm_expression (identifier) @property)
+((block_start) @punctuation)
+((block_end) @punctuation)
+
+(block (identifier) (identifier) @constructor)
+
+((plus_literal) @punctuation)
+((hash_literal) @comment)
+
+; relations
+((relation_literal) @function)
+(rel_expression (identifier) @property)
+
+
+((pipe_literal) @punctuation)
+
+(relation
+  (rel_expression
+    (
+  (hash_literal)
+  .
+  (identifier) @constant
+  ) @coment))
```

Comment:
stylings look very off here

Reply:
Apologies for not knowing much about these highlights files (this one comes from the repository of the treesitter implementation), but are you saying the formatting of this file is weird or that the highlight rules don't make sense?

Reply:
Only this specific query looks very off in styles. Others are perfectly fine

Something like 

```scheme
(relation
  (rel_expression
    (
      (hash_literal)
      .
      (identifier) @constant) @comment))
```

looks easier to one's eye imo